schema/workflow/unit/io/download_from_urls.json

```diff
@@ -0,0 +1,29 @@
+{
+    "schemaId": "workflow-unit-download-from-urls",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "download-from-urls io schema",
+    "type": "object",
+    "allof": [
+        {
+            "$ref": "../../unit.json"
+        }
+    ],
+    "properties": {
+        "url": {
```

Comment:
Replace with container information similar to 4309's PR (can probably be imported as a reference)

Reply:
The container has been replaced with an "ObjectProperties" object